bull-map-transformer/src/test/java/com/hotels/map/transformer/MapTransformerTest.java

```diff
@@ -236,11 +231,11 @@ public void testTransformWorksProperlyWithMapContainingList() {
         Map<MutableToFooSimple, List> actual = underTest.transform(sourceMap, MutableToFooSimple.class, List.class);
 
         //THEN
-        assertNotNull(actual);
-        assertEquals(actual.size(), sourceMap.size());
+        assertThat(actual).isNotNull();
+        assertThat(actual.size()).isEqualTo(sourceMap.size());
         for (Map.Entry<MutableToFooSimple, List> entry : actual.entrySet()) {
-            assertEquals(MutableToFooSimple.class, entry.getKey().getClass());
-            assertEquals(sampleList, entry.getValue());
+            assertThat(entry.getKey().getClass()).isEqualTo(MutableToFooSimple.class);
+            assertThat(entry.getValue()).isEqualTo(sampleList);
         }
```

Comment:
GIVEN block could be simplified to:

    import static org.assertj.core.util.Lists.list;
    import static org.assertj.core.util.Maps.newHashMap;
    ...

    Map<FromFooSimple, List<String>> sourceMap = newHashMap(fromFooSimple, list(ITEM_1));

Assertions could be simplified to:

    assertThat(actual).allSatisfy((key, value) -> {
        assertThat(key).isInstanceOf(MutableToFooSimple.class);
        assertThat(value).containsOnly(ITEM_1);
    });